src/RegExLib.Core/Entities/Expression.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using RegExLib.Core.Events;
+using RegExLib.Core.Interfaces;
+using RegExLib.SharedKernel;
+
+namespace RegExLib.Core.Entities
+{
+    public class Expression : BaseEntity
+    {
+        public int AuthorId { get; set; }
+        public string Title { get; set; }
+        public string Pattern { get; set; }
+        public string Description { get; set; }
+        public Author Author { get; set; }
+
+        public Expression()
```

Comment:
parameterless ctor. Please see above comment for Author

Reply:
I changed it to this

```csharp
        public readonly int AuthorId;
        public readonly string Title;
        public readonly string Pattern;
        public readonly string Description;
        public readonly Author Author;

        public Expression(string title, string pattern, string description, Author author)
        {
            Title = title;
            Pattern = pattern;
            Description = description;
            Author = author;
            AuthorId = author.Id;
        }
```